.changeset/spicy-kangaroos-enjoy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@wso2is/console": patch
+"@wso2is/i18n": patch
```

Comment:
Let's patch bump myaccount as well, as i18n module is a dependent of myaccount package.

You can view the dependent packages of a specific package by running `pnpm nx graph` in the project root.

Reply:
Updated with [fd82055](https://github.com/wso2/identity-apps/pull/5816/commits/fd820556010c0f240bd695f0764bbe70b220d01c)